packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -201,28 +206,48 @@ module.exports = async function generatePages(sourceDir, docsDir, options, chang
     )
   );
 
-  // Merge both sets of page sections into a single array.
+
+  const addCmsdsLink = function (page) {
+    if (
+      page.source.path.includes('node_modules/@cmsgov/design-system-docs/src/pages/components') ||
+      page.source.path.includes('node_modules/@cmsgov/design-system-docs/src/pages/patterns')
+    ) {
+      page.cmsds = `https://design.cms.gov/${page.referenceURI}`;
+    }
+    return page;
+  };
+  // Get pages that come from core CMS Design System
```

Comment:
This comment isnt quite accurate either, I would remove it or clarify what this line is doing.